spread.yaml

```diff
@@ -286,6 +287,14 @@ suites:
      # shellcheck source=/dev/null
      source "$HOME/venv/snapcraft/bin/activate"
      pip install --upgrade pip wheel setuptools
+
+     # Install python-apt using pip 20.1, last known working version.
+     # https://github.com/pypa/pip/issues/8654
+     pip install --upgrade pip==20.1
```

Comment:
Instead of a stage-packages, do you know if we can lock the pip version used? This might be something we should be able to do with snapcraft